packages/api/src/sequelize/migrations/2024-05-20_00_update-facility-organization-type.ts

```diff
@@ -98,5 +94,43 @@ export const down: Migration = ({ context: queryInterface }) => {
       },
       { transaction }
     );
+
+    const [facilityResults] = await queryInterface.sequelize.query(
+      `select * from ${facilityTableName}`,
+      {
+        transaction,
+      }
+    );
+
+    //eslint-disable-next-line @typescript-eslint/no-explicit-any
+    for (const facility of facilityResults as any[]) {
+      const cwType = facility.cw_type;
+      const cqType = facility.cq_type;
+
+      //eslint-disable-next-line @typescript-eslint/no-explicit-any
+      let type: any = "initiator_and_responder";
+
+      if (cwType === "initiator_only" || cqType === "initiator_only") {
+        type = "initiator_only";
```

Comment:
nit: this string `initiator_only` is reused ~5 times in this file. I'd recommend creating a const for it

Reply:
Also, this logic is slightly diff than the one on `up`: this one might result in the type being `initiator_only` even if the OBO OID is not set.